zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -136,17 +133,36 @@ pub async fn run() -> Result<()> {
 
     zebrad.expect_stdout_line_matches("activating mempool")?;
 
-    // TODO: check that lightwalletd is at the tip using gRPC (#4894)
-    //
-    // If this takes a long time, we might need to check zebrad logs for failures in a separate thread.
-
     tracing::info!(
         ?lightwalletd_rpc_port,
         "connecting gRPC client to lightwalletd...",
     );
 
     let mut rpc_client = connect_to_lightwalletd(lightwalletd_rpc_port).await?;
 
+    // Check if zebrad and lightwalletd are both in the same height.
+
+    // Get the block tip from lightwalletd
+    let block_tip = rpc_client
+        .get_latest_block(wallet_grpc::ChainSpec {})
+        .await?
+        .into_inner();
+
+    // Get the block tip from zebrad
+    let client = reqwest::Client::new();
+    let res = client
+        .post(format!("http://{}", &zebra_rpc_address.to_string()))
+        .body(r#"{"jsonrpc": "2.0", "method": "getblockchaininfo", "params": [], "id":123 }"#)
+        .header("Content-Type", "application/json")
+        .send()
+        .await?
+        .text()
+        .await?;
+
+    // Make sure tip from lightwalletd is the same as zebrad
+    let parsed: serde_json::Value = serde_json::from_str(&res)?;
+    assert_eq!(block_tip.height, parsed["result"]["blocks"]);
```

Comment:
We can move this into its own function and call it from `wait_for_zebrad_and_lightwalletd_sync`.

Here we'll want `wait_for_zebrad_tip = false`.